### ISSUE TITLE
This is not an issue , its a question

### ISSUE THREAD

**Issue by kageeker** [cloudwebrtc/flutter-webrtc-demo#2]
Is it possible to apply this on some other open source Webrtc servers like Kurento for example?

**Comment by cloudwebrtc** [comment]
Yes, you can try to connect to any other open source server, just write a signaling to communicate with it.
But now we don't have enough free time to write demo. The main work is still to implement flutter-webrtc.
